### ISSUE TITLE
[Issue] Licensing Issue

### ISSUE THREAD

**Issue by Dragon-Seeker** [vampire-studios/Obsidian#6]
This mod takes a lot of code from multiple places with some of it containing the original license while others do not. There are bits from the Mod DynReg that are here too. It would be great to properly give attribution and licensing for such.

**Comment by oliviathevampire** [comment]
Yes! I will hopefully be fixing that really soon! Working on cleaning up a lot of the code currently! Most of the code from DynReg will be gone asap as it is no longer needed!

**Comment by Mrthomas20121** [comment]
I was very sad to learn this mod took code from json things(which does literally the same thing as this mod but on forge/neoforge)

**Comment by oliviathevampire** [comment]
> I was very sad to learn this mod took code from json things(which does literally the same thing as this mod but on forge/neoforge)

I've been slowly working on removing the tiny amount of code I have from it
